packages/base/catalog-entry.gts

```diff
@@ -3,73 +3,61 @@ import {
   field,
   Component,
   CardDef,
-  FieldDef,
   relativeTo,
-  realmInfo,
+  linksToMany,
 } from './card-api';
 import StringField from './string';
 import BooleanField from './boolean';
 import CodeRef from './code-ref';
+import MarkdownField from './markdown';
 
-import { FieldContainer } from '@cardstack/boxel-ui/components';
 import GlimmerComponent from '@glimmer/component';
 import BoxModel from '@cardstack/boxel-icons/box-model';
+import BookOpenText from '@cardstack/boxel-icons/book-open-text';
+import LayersSubtract from '@cardstack/boxel-icons/layers-subtract';
+import GitBranch from '@cardstack/boxel-icons/git-branch';
+import { DiagonalArrowLeftUp } from '@cardstack/boxel-ui/icons';
+import { Pill } from '@cardstack/boxel-ui/components';
+import StackIcon from '@cardstack/boxel-icons/stack';
+import AppsIcon from '@cardstack/boxel-icons/apps';
+import LayoutList from '@cardstack/boxel-icons/layout-list';
+import Brain from '@cardstack/boxel-icons/brain';
+
+export class SpecType extends StringField {
+  static displayName = 'Spec Type';
+}
 
 export class CatalogEntry extends CardDef {
   static displayName = 'Catalog Entry';
   static icon = BoxModel;
-  @field title = contains(StringField);
-  @field description = contains(StringField);
+  @field name = contains(StringField);
+  @field readMe = contains(MarkdownField);
+
   @field ref = contains(CodeRef);
+  @field type = contains(SpecType);
 
-  // If it's not a field, then it's a card
-  @field isField = contains(BooleanField);
+  @field isField = contains(BooleanField, {
+    computeVia: function (this: CatalogEntry) {
+      return this.type === 'field';
+    },
+  });
 
+  @field isCard = contains(BooleanField, {
+    computeVia: function (this: CatalogEntry) {
+      return this.type === 'card';
+    },
+  });
   @field moduleHref = contains(StringField, {
     computeVia: function (this: CatalogEntry) {
       return new URL(this.ref.module, this[relativeTo]).href;
     },
   });
-  @field demo = contains(FieldDef);
-  @field realmName = contains(StringField, {
+  @field examples = linksToMany(CardDef);
+  @field title = contains(StringField, {
     computeVia: function (this: CatalogEntry) {
-      return this[realmInfo]?.name;
+      return this.name || this.ref.name;
     },
   });
```

Comment:
This won't work well for default exports. The ref name will be 'default' for each one